File: java/Kode/WeaponMaterial.java
public class WeaponMaterial {
    private String rarity;
    private String quality;

    public WeaponMaterial(String rarity, String quality) {
        this.rarity = rarity;
        this.quality = quality;
    }

    public void displayMaterial() {
        System.out.println("  [MATERIAL] Rarity: " + rarity + ", Quality: " + quality);
    }
}

File: cpp/Kode/WeaponMaterial.cpp
#include "WeaponMaterial.h"

WeaponMaterial::WeaponMaterial(std::string r, std::string q) : 
    Rarity(r), Quality(q) {}

void WeaponMaterial::displayMaterial() const {
    std::cout << "  [MATERIAL] Rarity: " << Rarity << ", Quality: " << Quality;
}